### ISSUE TITLE
some question About diffusion Model

### ISSUE THREAD

**Issue by zshy1205** [adelacvg/ttts#13]
about the  diffusion model input，why use gpt latent？
if use vq code as diffusion model input，is something wrong?
compare with spear tts，this model vq code is semantic token?
thanks

**Comment by adelacvg** [comment]
According to the Tortoise paper, the latent representation of GPT performs better than the vq code.

**Comment by zshy1205** [comment]
thanks，I read the Tortoise paper，the paper is “For the majority of the training procedure, the DDPM is trained to convert discrete speech codes into MEL spectrograms. After this process has converged, I fine-tune the DDPM on the autoregres-
sive latent space which is pulled from the AR model outputs instead of the speech codes‘
so，If I try train DDPM, I can train the ddpm with vq token as input，and if the AR model was trained，than I should finetune DDPM with AR latent.

**Comment by Liujingxiu23** [comment]
@zshy1205  Have you train the diffusion part ? How about the loss and the final performace ?
